comfy_cli/command/knowledge.py

```
@@ -48,7 +47,7 @@ def _require_bundle(renderer) -> knowledge.Bundle:
         renderer.error(
             code="knowledge_unavailable",
             message="no knowledge bundle is loaded",
-            hint="set COMFY_KNOWLEDGE_FILE to a knowledge.json, or COMFY_KNOWLEDGE_URL to fetch one; see `comfy knowledge status`",
+            hint="sign in with `comfy cloud login` so the bundle can be fetched, or set COMFY_KNOWLEDGE_FILE to a knowledge.json; see `comfy knowledge status`",
```

Pylint (4.0.7)

[convention] Line too long (160/120)

(C0301)